spyderlib/utils/icon_manager.py

```diff
@@ -1,8 +1,16 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2015 The Spyder Development Team
```

Comment:
This line must read

    Copyright © 2009- The Spyder Development Team

Please change it